src/domain/AOReward.ts

```diff
@@ -0,0 +1,78 @@
+import { BigInt, BigDecimal, log } from '@graphprotocol/graph-ts'
+import { Pool, PoolAddresses, RewardDayTotal, RewardLink, AORewardBalance } from '../../generated/schema'
+import { aoRewardsCeiling, secondsInDay } from '../config'
+import { loadOrCreateRewardDayTotal } from './Reward'
+
+export function loadOrCreateAORewardBalance(address: string): AORewardBalance {
+  let rb = AORewardBalance.load(address)
```

Comment:
Would rather more descriptive variable names 😄

Reply:
Yeah, this is an artifact from an earlier version – therefore not changing it here now :D